docs/developer-guide/working-with-rules.md

```diff
@@ -74,7 +74,7 @@ The `context` object contains additional functionality that is helpful for rules
 * `id` - the rule ID.
 * `options` - an array of rule options.
 * `settings` - the `settings` from configuration.
-* `parserName` - the `parser` from configuration.
+* `parserPath` - the full path to the `parser` from configuration.
```

Comment:
Changed from `parserName` to better reflect the value that's now there. (`parserName` was added in previous beta.)